code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,253 @@
+
+	max_boxes = 12
+
+	ego_list = list(
+		/datum/ego_datum/weapon/letter_opener
+		)
+	abnormality_origin = ABNORMALITY_ORIGIN_LOBOTOMY
+	var/REPRESSCOUNT = 0 // V causes painmode
+	var/painmode = FALSE //this refers to papercut damage increase when 3rd repression is done
+	var/AFFECTEDCELL = list() // turf with funny
+	var/cooldown
+	var/cooldown_time = 30 SECONDS
+
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/ZeroQliphoth(mob/living/carbon/human/user)
+	return
+
+//papercut due to failed work
+/mob/living/simple_animal/hostile/abnormality/mailpile/WorktickFailure(mob/living/carbon/human/user)
+	if(painmode)
+		user.apply_damage(50, RED_DAMAGE, user.run_armor_check(null, BRUTE), spread_damage = FALSE)
+		to_chat(user,"The paper tears through your flesh as you scream out in agony.")
+		return
+	user.apply_damage(5, RED_DAMAGE, user.run_armor_check(null, RED_DAMAGE), spread_damage = FALSE)
+	to_chat(user,"Ouch! I got a paper cut!")
+	return
+//players get a parcel / letters for completing works
+//represscount and painmode refer to special ego obtainment, 2 repressions cause pain mode, pain mode makes papercuts
+/mob/living/simple_animal/hostile/abnormality/mailpile/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(work_type == ABNORMALITY_WORK_REPRESSION)
+		if(cooldown > world.time)
+			to_chat(user,"You realized you have made a grave mistake as envelopes start flying out of the mailbox towards you.")
+			user.Stun(100)
+			var/letterssave = list()
+			for (var/i = 1 to 4)
+				var/obj/item/mailpaper/MAIL = new(get_turf(src))
+				MAIL.mailtype = "Hatred"
+				MAIL.name = "A hastily written angry letter"
+				MAIL.desc = "IHATEYOUIHATEYOUIHATEYOUIHATEYOUIHATEYOU."
+				MAIL.throw_at(user, 4, 1, src, spin = FALSE, gentle = TRUE, quickstart = FALSE)
+				letterssave += MAIL
+				sleep(10)
+			user.gib()
+			sleep(50)
+			for (var/obj/item/mailpaper/MAIL in letterssave)
+				qdel(MAIL)
+			return
+		cooldown = world.time + cooldown_time
+		REPRESSCOUNT++
+		if(REPRESSCOUNT == 2)
+			painmode = TRUE
+			for (var/i = 1 to 10)
+				var/turf/dispense_turf = get_step(src, i)
+				var/obj/effect/letters_flow/EFFECT = new /obj/effect/letters_flow(dispense_turf)
+				AFFECTEDCELL += EFFECT
+		if(REPRESSCOUNT == 3)
+			painmode = FALSE
+			var/obj/item/parcelself/PARCEL = new(get_turf(src))
+			PARCEL.desc = "This parcel seems to be addressed to [user.name]. It seems to have come from the future..?"
+			PARCEL.throw_at(user, 4, 1, src, spin = FALSE, gentle = TRUE, quickstart = FALSE)
+			PARCEL.receiver = user
+			REPRESSCOUNT = 0
+			for(var/obj/effect/letters_flow/EFFECT in AFFECTEDCELL)
+				EFFECT.Destroy()
+
+		return
```

Comment:
You have to sacrifice _3 PEOPLE_ for this?

Reply:
I've just realized this runs after the one that could gib you... Wait you have to repress this thing twice AFTER someone gibs to it for this??? Within 30 seconds? Including taking i think guaranteed 600 red damage?